src/Primitives/emulated.cpp

```diff
@@ -454,7 +460,7 @@ def_prim(subscribe_interrupt, threeToNoneU32) {
 }
 
 // Temporary Primitives needed for analogWrite in ESP32
-def_prim(chip_analog_write, threeToNoneU32) {
+def_prim(chip_ledc_set_duty, threeToNoneU32) {
```

Comment:
This primitive is no longer needed

Reply:
Maybe developers want both of them? They are slightly different.

Reply:
indeed good idea